library/CLAUDE.md

```
@@ -0,0 +1,120 @@
+# library - Chucker Core Library
+
+The main module. Contains the OkHttp interceptor, Room database, and Android UI for HTTP inspection.
+
+## Architecture
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)